velox/benchmarks/SimpleArithmetic.cpp

```diff
@@ -15,13 +15,11 @@
  */
 
 #include <folly/Benchmark.h>
-#include <folly/init/Init.h>
```

Comment:
These changes seem unrelated to this diff. It would be nice to move them into a separate PR.